manifests/4.9/kube-descheduler-operator.crd.yaml

```diff
@@ -92,6 +92,9 @@ spec:
                   - TopologyAndDuplicates
                   - LifecycleAndUtilization
                   - DevPreviewLongLifecycle
+                  - SoftTopologyAndDuplicates
+                  type: string
```

Comment:
duplicated line

Reply:
Sorry, which line is this a duplicate of?

Reply:
Ah I see, it's above on L89... weird, this was generated

Reply:
Gotcha